src/ngx_http_vhost_traffic_status_string.c

```diff
@@ -164,4 +164,66 @@ ngx_http_vhost_traffic_status_replace_strc(ngx_str_t *buf,
     return NGX_OK;
 }
 
+
+ngx_int_t
+ngx_hex_escape_invalid_utf8_char(ngx_pool_t *pool, ngx_str_t *buf, u_char *p, size_t n)
```

Comment:
Since this also escapes double-quotes (and possibly other values - see below), maybe call it `ngx_http_vhost_traffic_status_escape_prometheus_pool` to match it's JSON equivalent - https://github.com/vozlt/nginx-module-vts/blob/46d85558e344dfe2b078ce757fd36c69a1ec2dd3/src/ngx_http_vhost_traffic_status_string.c#L71